addons/account_edi_ubl_cii/models/res_partner.py

```diff
@@ -12,11 +12,11 @@ class ResPartner(models.Model):
         string="Format",
         selection=[
             ('facturx', "Factur-X (CII)"),
-            ('xrechnung', "XRechnung (UBL)"),
-            ('nlcius', "NLCIUS"),
-            ('ubl_bis3', "Peppol Bis Billing 3.0"),
-            ('ubl_a_nz', "A-NZ BIS Billing 3.0"),
-            ('ubl_sg', "SG BIS Billing 3.0"),
+            ('ubl_bis3', "Peppol Bis 3.0 UBL"),
+            ('xrechnung', "Peppol Bis 3.0 UBL (XRechnung)"),
+            ('nlcius', "Peppol Bis 3.0 UBL (NLCIUS)"),
+            ('ubl_a_nz', "Peppol Bis 3.0 UBL (A-NZ)"),
+            ('ubl_sg', "Peppol Bis 3.0 UBL (SG)"),
```

Comment:
But that's not true :sweat: This is about the format, not the web-service. You will be able to use this module without yours later so that's not true to say "Peppol" here...